src/core/CoolBoard.cpp

```diff
@@ -589,8 +641,13 @@ void CoolBoard::mqttLog(String data) {
   }
 }
 
-bool CoolBoard::mqttPublish(String data) {
-  return (this->coolPubSubClient->publish(this->mqttOutTopic.c_str(),
+bool CoolBoard::mqttPublish(String data, bool mpack) {
```

Comment:
so I gather there is still backward compatibility with plain JSON messages, but is it still used within that firmware ?

Reply:
I use Json message only for update `"desired":{}`